bot/exts/info/resources.py

```diff
@@ -0,0 +1,79 @@
+from typing import Optional
+
+import requests
+from bs4 import BeautifulSoup
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+RESOURCES_URL = "https://www.pythondiscord.com/resources/"
+SITE_TOPICS_TITLE_CASE = []
+
+
+def get_topics() -> None:
+    """Scrape Python Discord website for updated list of topics to include in embed."""
+    topics_split = []
+    topics_title = ""
+    dont_capitalize = ["for", "and", "nor", "but", "or", "yet", "so", "in", "of", "a", "an"]
+    page = requests.get(RESOURCES_URL)
+    soup = BeautifulSoup(page.content, 'html.parser')
+    # Obtain the topics using CSS selector
+    topics_initial = soup.select('label.checkbox input[data-filter-name="topics"]')
+    # Split topics at the dash, add to list, prep for iteration over each word
+    for topic in topics_initial:
+        topics_split.append(topic['data-filter-item'].split("-"))
+    # Iterate over each word in order to title case and remove the OTHER category (not helpful to include in embed)
+    for topic in topics_split:
+        for word in range(0, len(topic)):
+            if topic[word] != "other":
+                if topic[word] not in dont_capitalize:
+                    topics_title += topic[word].title() + " "
+                if topic[word] in dont_capitalize:
+                    topics_title += topic[word] + " "
+                if word == len(topic) - 1:
+                    SITE_TOPICS_TITLE_CASE.append(topics_title)
+                    topics_title = ""
```

Comment:
Great work with this, but I agree with the comment made by @Shivansh-007 [in the original issue](https://github.com/python-discord/bot/issues/2079#issuecomment-1070529971) that [site](https://github.com/python-discord/site) should get an API such that we do not need to do this scraping.

I think for now, considering what you may be down to do, we can remove this and open another issue to improve it later once the site has received this new API. What do you think?

Reply:
Sure... I am excited to help build the API. I've never done that before, nor do I really know what it would provide, but I'm totally down to help out. Without looking into it, I assume it means that there would be some sort of interface provided to extract the information in an easier way?

Reply:
@Bluenix2 - could you open the issue for creating an API to handle the scraping? It would be really exciting to have your guidance as I work through my first experience with APIs.